docker-compose/dse-6.8/docker-compose.yml

```diff
@@ -3,9 +3,10 @@ version: '2'
 services:
   dse-1:
     image: datastax/dse-server:${DSETAG}
+    platform: linux/amd64
```

Comment:
enforcing platform architecture since this is the only architecture currently supported for DSE. Users will likely also require enablement of Docker's Rosetta emulation features to run this image on M1 macs.